concepts/0441-present-proof-best-practices/README.md

```diff
@@ -149,6 +149,18 @@ A presentation may include a timestamp outside of a the non-revocation interval
 
 Any timestamp in the future (relative to the instant of presentation proof, with a reasonable allowance for clock skew) evinces tampering: the verifier MUST reject a presentation with a future timestamp. Similarly, any timestamp predating the creation of its corresponding credential's revocation registry on the ledger evinces tampering: the verifier MUST reject a presentation with such a timestamp.
 
+### Dates and Predicates
+
+This section prescribes issuer and verifier best practices concerning representing dates for use in predicate proofs (eg proving Alice is over 21 without revealing her birth date).
+
+#### Dates in Credentials
+
+In order for dates to be used in a predicate proof they MUST be expressed as an Int32. While unix timestamps could work for this, it has several drawbacks including: can't represent dates outside of the years 1901-2038, isn't human readable, and is overly percise in that birth time down to the second is generally not needed for an age check. To address these issues, dates attributes SHOULD be represented as integers in the form YYYYMMDD. This addresses the issues with unix timestamps (or any seconds-since-epoch system) while still allowing date values to be compared with < > operators. Note that this system won't work for any general date math (eg adding or subtracting days), but it will work for predicate proofs which just require comparisons. In order to make it clear that this format is being used, the attribute name SHOULD have the suffix `_dateint`. Since most datetime libraries don't include this format, [here](https://github.com/kiva/protocol-common/blob/main/src/date.conversion.ts) are some examples of helper functions written in typescript.
```

Comment:
Typo percise vs. precise 

Ironic, don't you think?

Reply:
Haha, good catch!